Debian/8/templates/csv/mount_options.csv

```diff
@@ -0,0 +1,7 @@
+# mountpoint options that should be set
+/home,nosuid nodev
+/var/log,nosuid nodev noexec
+/var/lib,nodev nosuid
+/var,nosuid nodev noexec
+/tmp,nodev nosuid
```

Comment:
@pthierry38 I think that we should make every option comma separated. Then if the option exists, create the OVAL.